app/aws/s3.py

```diff
@@ -66,6 +70,35 @@ def get_job_from_s3(service_id, job_id):
     return obj.get()["Body"].read().decode("utf-8")
 
 
+def get_phone_number_from_s3(service_id, job_id, job_row_number):
+    # We don't want to constantly pull down a job from s3 every time we need a phone number.
+    # At the same time we don't want to store it in redis or the db
+    # So this is a little recycling mechanism to reduce the number of downloads.
+    job = JOBS.get(job_id)
+    if job is None:
+        job = get_job_from_s3(service_id, job_id)
+        JOBS[job_id] = job
+    job = job.split("\r\n")
+    first_row = job[0]
+    job.pop(0)
+    first_row = first_row.split(",")
+    phone_index = 0
+    for item in first_row:
+        if item == "phone number":
+            break
+        phone_index = phone_index + 1
+    correct_row = job[job_row_number]
+    correct_row = correct_row.split(",")
+
+    my_phone = correct_row[phone_index]
+    my_phone = my_phone.replace("(", "")
+    my_phone = my_phone.replace(")", "")
+    my_phone = my_phone.replace("-", "")
+    my_phone = my_phone.replace(" ", "")
+    my_phone = my_phone.replace("+", "")
```

Comment:
In the case of the `+` symbol, will that be okay if it is included?  This assumes a country code is there as well and it'd end up with a number like `15555555555` - totally fine if that's intended!  I see it in the test cases but just wanted to be sure.

Should we also account for `.` in the phone number (e.g., 555.555.5555)?  I realize that's not typically seen in the US, but it does exist.

Ultimately I'm assuming the formatting of the phone number has to match what we were retrieving originally.

Additionally, I was going to suggest that we could use the `re` module to take care of the substitution in one line.  As a contrived example that goes through a few different phone number formatting variations:

```python
import re

phone_numbers = [
    '+1 (555) 555-5555',
    '555-555-5555',
    '(555) 555-5555',
    '555.555.5555',
    '+1.555.555.5555',
]

for phone_number in phone_numbers:
    re.sub(r'[\+\s\(\)\-\.]*', '', phone_number)

# '15555555555'
# '5555555555'
# '5555555555'
# '5555555555'
# '15555555555'
```

But I'm not sure if that's more "readable" so to speak... 😆 - I leave it up to you! 🙂

Reply:
Yeah, readability versus 'the right way', always a problem.  I'm fine to do it your way.

Reply:
I'm not sure reaching for the `re` library could be considered the "right" way either! 😆